src/view/container/device/Device.tsx

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import Simulator from "../../components/Simulator";
```

Comment:
nit: I see in inconsistency in imports formatting, worth using some extension or setting

Reply:
awesome, thanks for the comment. I will look into the current/lint format tool

Reply:
I have updated the tslint to order the imports